ui/components/MesheryMeshInterface/PatternService/RJSF.js

```diff
@@ -82,7 +83,7 @@ function RJSFForm({
           CheckboxWidget : CustomCheckboxWidget,
           TextareaWidget : CustomTextAreaWidget,
         }}
-        liveValidate
+        liveValidate={liveValidate}
```

Comment:
@Abhishek-kumar09 @harkiratsm I disabled the liveValidate here so that it will not show red fields, I need a small help is that after disabling liveValidate I am not getting any error info from prop ` transformErrors`

Though according to docs disabling liveValidate mean it does validation after submit event so, I am stuck here how that will show on field

Request here is user should only be able to submit data only when there is no error

Reply:
Why not live validate ?

Reply:
It is important that users are aware of which fields are required or they are filling out something incorrectly.

Let's catch up today and complete this.

Reply:
Thanks sure I'll ping you